packages/xdl/src/Env.ts

```diff
@@ -27,3 +27,7 @@ export function maySkipManifestValidation(): boolean {
 export function shouldUseDevServer(exp: ExpoConfig) {
   return Versions.gteSdkVersion(exp, '40.0.0') || getenv.boolish('EXPO_USE_DEV_SERVER', false);
 }
+
+export function isE2EOrCI() {
```

Comment:
maybe sth less specific would be cleaner, e.g shouldEnableAnalytics,shouldEnableTracking

Reply:
I do like keeping it generic so other places in the CLI can also use this function to disable/enable behaviour on CI, but I'm happy to make the change. we can always revert it later on if we need to use it somewhere else.